package/MDAnalysis/analysis/dihedrals.py

```diff
@@ -89,6 +124,16 @@
        an n_frames×n_residues×2 :class:`numpy.ndarray` with content
        ``[[[phi, psi], [residue 2], ...], [time step 2], ...]``.
 
+.. autoclass:: Janin
+   :members:
+   :inherited-members:
+
+   .. attribute:: angles
+
+       Contains the time steps of the phi and psi angles for each residue as
```

Comment:
chi1 and chi2

Btw, you can use proper Greek characters with LaTeX-like formatting
```reST
contains :math:`\chi_` and :math:`\chi_2` angles
```